scripts/opencontrail-kubelet/opencontrail_kubelet/plugin.py

```diff
@@ -231,4 +240,4 @@ def main():
         sys.exit(1)
 
 if __name__ == '__main__':
-    main()
+        main()
```

Comment:
Oops, sorry. That is a good catch. Did not realize..

Thx much.

Regards,
Ananth

On Wed, Jul 8, 2015 at 4:44 PM, Pedro Marques <notifications@github.com>
wrote:

> In scripts/opencontrail-kubelet/opencontrail_kubelet/plugin.py
> <https://github.com/Juniper/contrail-kubernetes/pull/13#discussion_r34211146>
> :
>
> > @@ -231,4 +240,4 @@ def main():
> >          sys.exit(1)
> >
> >  if __name__ == '__main__':
> > -    main()
> > +        main()
>
> Sorry. Indentation is incorrect. Just one more tiny change.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Juniper/contrail-kubernetes/pull/13/files#r34211146>.
>